proxy/coco/src/peer/run_state.rs

```diff
@@ -199,29 +284,59 @@ impl From<Config> for RunState {
             config,
             connected_peers: HashSet::new(),
             status: Status::Stopped(Instant::now()),
+            waiting_room: WaitingRoom::new(waiting_room::Config::default()),
```

Comment:
The default here will set the delta to `0` because I believe that's the default for `Duration`. We should probably define `Duration` that makes more sense as a `const` to set it as, or pass it from the `Config`.

Reply:
fe90e77